tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateAddStdUserAsProjectOwner() {
+func (rb *RBACAdditionalTestSuite) TestRbacP1ClusterOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	clusterAdmin, standardUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	rb.standardUserCOProject = createProjectAsCO
 
-	log.Info("Validating if cluster owner can add a user as project owner in a project")
-	err = users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner, nil)
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	userGetProject, err := projects.GetProjectList(rb.additionalUserClient, rb.cluster.ID)
+
+	rb.T().Logf("Adding user as " + roleOwner + " to the downstream cluster.")
+	err = users.AddClusterRoleToUser(rb.client, rb.cluster, clusterAdmin, roleOwner, nil)
+	require.NoError(rb.T(), err)
+	standardUserClient, err = standardUserClient.ReLogin()
 	require.NoError(rb.T(), err)
-	assert.Equal(rb.T(), 1, len(userGetProject.Data))
-	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
 
-	err = users.RemoveProjectMember(rb.standardUserClient, rb.additionalUser)
+	rb.T().Logf("Validating if " + roleOwner + " can add another standard user as a project owner")
+	clusterOwnerProject, err := createProject(rb.client, rb.cluster.ID)
 	require.NoError(rb.T(), err)
-}
+	err = users.AddProjectMember(standardUserClient, clusterOwnerProject, additionalUser, roleProjectOwner, nil)
+	require.NoError(rb.T(), err)
+	projectListClusterAdditionalUser, err := projects.ListProjectNames(additionalUserClient, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	projectListClusterAdmin, err := projects.ListProjectNames(standardUserClient, rb.cluster.ID)
```

Comment:
in general, can you add some new lines after the last require.* or assert.* in a block? This will help with readability.